crates/store/re_data_loader/src/loader_rrd.rs

```diff
@@ -201,24 +196,23 @@ impl Read for RetryableFileReader {
 
 impl RetryableFileReader {
     fn block_until_file_changes(&self) -> std::io::Result<usize> {
+        #[allow(clippy::disallowed_methods)]
```

Comment:
not too happy about this, but saying that this is only used for non-wasm target opens 10 more 5 more places with issues